lib/main.js

```diff
@@ -16,16 +16,17 @@ var __importStar = (this && this.__importStar) || function (mod) {
     return result;
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-const core = __importStar(require("@actions/core"));
 const exec = __importStar(require("@actions/exec"));
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         try {
-            // await exec.exec('julia', ['--color=yes', '-e', 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'])
-            yield exec.exec('julia', ['--color=yes', '-e', 'using Pkg; Pkg.add(PackageSpec(name="Coverage", rev="master")); using Coverage; Coveralls.submit(process_folder())']);
+            yield exec.exec('julia', ['--color=yes', '-e', 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())']);
         }
         catch (error) {
-            core.setFailed(error.message);
+            // We are making errors non fatal for now because this for example
+            // will generally not work if triggered by a PR, which is a common
+            // situation.
+            // core.setFailed(error.message)
```

Comment:
Perhaps this could be the decision of the user by setting [`jobs.<job_id>.steps.continue-on-error`](https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) in the workflow?

If not, let's at least log the error message to allow debugging if the failure is unexpected:

```suggestion
            // core.setFailed(error.message)
            core.debug(error.message)
```